plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java

```diff
@@ -123,6 +126,12 @@ public Builder setSchemaInitializer(SchemaInitializer schemaInitializer)
             return self();
         }
 
+        public Builder addAdditionalCatalog(Plugin connectorPlugin, String catalogName, String connectorName)
```

Comment:
if we wanted to have this, this would go into parent DQR, not specific to iceberg.

for now let's remove it, see comment in TIV2.